pandas/core/series.py

```diff
@@ -394,6 +401,11 @@ def __init__(
                 self.name = name
             return
 
+        if isinstance(data, (ExtensionArray, np.ndarray)):
+            if default_cow_copy and not copy and using_copy_on_write():
```

Comment:
i find this difficult to reason about (... in part bc im distracted by being on a call).  if a user explicitly passes `copy=False`, we should never ignore that.  Does this (or any of the other CoW-constructor PRs that im having trouble keeping track of) respect that?

Reply:
If you pass an array then we respect copy=False with the drawback that your series gets modified when you modify the array, if you pass a Series/DataFrame we make a lazy copy to set up references

Reply:
This is actually the reason why I changed the default to None, so that we can respect a user actually passing False

Reply:
I also don't find it easy to read, but I think it can be simplified? 
Wouldn't it be enough to check for if `copy` was originally None? Because if `copy=True`, I would expect the copy to happen anyway later on (assuming it currently honored that keyword)